contracts/Bridge/XVSBridgeAdmin.sol

```diff
@@ -0,0 +1,74 @@
+// SPDX-License-Identifier: BSD-3-Clause
+
+pragma solidity 0.8.13;
+
+import { AccessControlledV8 } from "@venusprotocol/governance-contracts/contracts/Governance/AccessControlledV8.sol";
+
+interface IXVSBridge {
+    function transferOwnership(address addr) external;
+}
+
+contract XVSBridgeAdmin is AccessControlledV8 {
+    /// @custom:oz-upgrades-unsafe-allow state-variable-immutable
+    IXVSBridge public immutable XVSBridge;
+
+    mapping(bytes4 => string) public functionRegistry;
+
+    /// @custom:oz-upgrades-unsafe-allow constructor
+    constructor(address XVSBridge_) {
+        require(XVSBridge_ != address(0), "Address must not be zero");
+        XVSBridge = IXVSBridge(XVSBridge_);
+        _disableInitializers();
+    }
+
+    /**
+     * @notice Invoked when called function does not exist in the contract
+     */
+    fallback() external {
+        string memory res = _getFunctionName(msg.sig);
+        require(bytes(res).length != 0, "Function not found");
+        _checkAccessAllowed(res);
+        (bool ok, ) = address(XVSBridge).call(msg.data);
+        require(ok, "call failed");
```

Comment:
Shouldn't we return the result of the call?

Should we "propagate" the `msg.value` just in case the target function needs it?

```suggestion
        (bool ok, bytes memory res) = address(XVSBridge).call{ value: msg.value }(msg.data);
        require(ok, "call failed");
        return res;
```

Reply:
Fixed [here](https://github.com/VenusProtocol/isolated-pools/pull/291/commits/bed382163a59d6ed783b0459a89c34fd3599771e)